MK2CHOPR/MK2CHOPR-IOC-01App/Db/MK2CHOPR.db

```diff
@@ -0,0 +1,391 @@
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(bi, "$(P)CHOP:STAT:CLOCK") {
+  field(DESC, "Main clock loss")
+  field(INP, "$(P)CHOP:STAT.B1 CP")
+  field(ZNAM, "OK")
+  field(ONAM, "Clock loss")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(bi, "$(P)CHOP:STAT:B1:OVERHEAT") {
+  field(DESC, "Bearing 1 is overheating")
+  field(INP, "$(P)CHOP:STAT.B2 CP")
+  field(ZNAM, "OK")
+  field(ONAM, "Overheat")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(bi, "$(P)CHOP:STAT:B2:OVERHEAT") {
+  field(DESC, "Bearing 2 is overheating")
+  field(INP, "$(P)CHOP:STAT.B3 CP")
+  field(ZNAM, "OK")
+  field(ONAM, "Overheat")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(bi, "$(P)CHOP:STAT:MOT:OVERHEAT") {
+  field(DESC, "Motor is overheating")
+  field(INP, "$(P)CHOP:STAT.B4 CP")
+  field(ZNAM, "OK")
+  field(ONAM, "Overheat")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(bi, "$(P)CHOP:STAT:SPEED") {
+  field(DESC, "Chopper is overspeed")
+  field(INP, "$(P)CHOP:STAT.B5 CP")
+  field(ZNAM, "OK")
+  field(ONAM, "Overspeed")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+#######################################################
+# Spectral locks
+#
+# Note that this varies with chopper type. The protocol
+# is for an INDRA* type chopper
+#######################################################
+
+record(mbbiDirect, "$(P)SPEC:STAT") {
+  field(DESC, "Spectral interlock status")
+  field(SDIS, "$(P):DISABLE")
+  field(SCAN, "1 second")  
+  field(DTYP, "stream")
+  field(INP, "@mk2chopper.proto getSpectralLocks $(PORT)")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:SPEC:STAT")
+}
+
+record(mbbiDirect, "$(P)SIM:SPEC:STAT")
+{
+}
+
+record(bi, "$(P)SPEC:STAT:DRIVE") {
+  field(DESC, "Wheter the drive is running")
+  field(INP, "$(P)SPEC:STAT.B0 CP")
+  field(ZNAM, "Not running")
+  field(ONAM, "Running")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(bi, "$(P)SPEC:STAT:REG") {
+  field(DESC, "Wheter the system is in reg. mode")
+  field(INP, "$(P)SPEC:STAT.B1 CP")
+  field(ZNAM, "Not Reg. mode")
+  field(ONAM, "Reg. mode")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+record(bi, "$(P)SPEC:STAT:SYNC") {
+  field(DESC, "Wheter the system is in sync")
+  field(INP, "$(P)SPEC:STAT.B2 CP")
+  field(ZNAM, "No sync")
+  field(ONAM, "Sync")
+  info(INTEREST, "HIGH")
+  info(archive, "VAL")
+}
+
+#######################################################
+# Error flags
+#######################################################
+
+record(mbbiDirect, "$(P)ERR:STAT") {
+  field(DESC, "Error states")
+  field(SDIS, "$(P):DISABLE")
+  field(SCAN, "1 second")  
+  field(DTYP, "stream")
+  field(INP, "@mk2chopper.proto getErrors $(PORT)")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:ERR:STAT")
+}
+
+record(mbbiDirect, "$(P)SIM:ERR:STAT")
+{
+}
+
+record(bi, "$(P)ERR:PHASE:DELAY") {
+  field(DESC, "The phase delay is wrong for the current speed")
```

Comment:
Will look into this. Those types of errors are often a red herring for something else